src/GT/Strava.hs

```diff
@@ -0,0 +1,97 @@
+{-# LANGUAGE RankNTypes      #-}
+{-# LANGUAGE RecordWildCards #-}
+
+-- | Import data from Strava.
+--
+-- This module uses the @Strive@ library.
+module GT.Strava where
+
+import GT.Prelude
+
+import           Control.Monad.Logger
+import           Data.Maybe
+import           Data.String
+import qualified Data.Text            as Text
+import           Database.Persist.Sql as DB
+import           Strive               hiding
+    ( map
+    )
+
+import qualified GT.DB.Schema.StravaActivity as DB
+import qualified GT.DB.Schema.StravaGear     as DB
+
+-- | Scaffolding. Ignore this.
+type Runner = forall x m. SqlPersistT m x -> m x
```

Comment:
I don't want to bother setting up the actual database running stuff, so I create this type alias which 'throws away' the `SqlPersistT` business from `persistent`. It's a throw-away bit of code that'll only be used to get this type checking.